kparks29/stocktracker
# Fix gulp testand sass

@MajorMetal @samjwalston 
Checkout my pull request.  I want you to get familiar with my code changes as well.
Please comment on this to let me know you seen it so I can merge this.

DESCRIPTION
Fix 'gulp sass' to watch all .scss files
Fix 'gulp test' to run test properly
Add pre push ghook, this runs test before pushing to github

HOW TO TEST
1) pull latest changes
2) run 'npm install' and 'gulp build' (if still having trouble delete node modules folder and repeat)
3) open browser
4) change something in _login.scss
5) notice it auto updates in browser
6) change a file, and run 'git add .' & 'git commit' & 'git push' command
7) notice that it will run tests on the git push (it also creates a coverage report in a new folder called coverage)
